src/submodules/schedulers.jl

```diff
@@ -1,17 +1,11 @@
 export schedule, Schedulers
 
 """
-    schedule(model [, scheduler]) → ids
+    schedule(model::StandardABM) → ids
 
-If no `scheduler` is given, it returns an iterator over the scheduled IDs using the model's
-scheduler, otherwise it uses the given custom scheduler, which can be either a function which
-accepts `model` as argument or one of the already defined schedulers inside Agents.jl. See
-the [manual scheduling](@ref manual_scheduling) section for usage examples.
+Return an iterator over the scheduled IDs using the model's default scheduler.
 """
-schedule(model::ABM) = schedule(model, abmscheduler(model))
-schedule(model::ABM, scheduler) = Iterators.filter(id -> id in allids(model), scheduler(model))
```

Comment:
We would need to keep this line instead of `schedule(model::ABM, scheduler) = scheduler(model)` to deprecate this properly (also passing a deprecation warning probably)